packages/frontend/component/src/lit-react/lit-portal/lit-portal.tsx

```diff
@@ -7,12 +7,11 @@ import ReactDOM from 'react-dom';
 type PortalEvent = {
   name: 'connectedCallback' | 'disconnectedCallback' | 'willUpdate';
```

Comment:
maybe `firstUpdated` would be better

Reply:
what is the use case of `firstUpdated` event?